examples/bank/bank.go

```diff
@@ -27,6 +27,8 @@ import (
 	"time"
 
 	"github.com/cockroachdb/cockroach/client"
+	// register HTTP sender for use in client.Open
+	_ "github.com/cockroachdb/cockroach/client/httpsender"
```

Comment:
I don't like that every client app will need to add a useless import somewhere. I think it would be better to leave httpsender in the main client package (which is fine since it doesn't have any non-standard dependencies) and only move rpcsender.